app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
Please convert all html files to slim. You can use such html like `https://www.html2slim.com/`. Do not forget to change extensions to `.slim`